doc/nrf/config_and_build/bootloaders/index.rst

```diff
@@ -51,12 +51,17 @@ You can find an overview of currently supported bootloaders in the table below:
      - Yes
      - :doc:`See imgtool <mcuboot:imgtool>`
      - No
-     - Yes
+     - Yes `*`
      - Yes
      - :ref:`Monotonic (HW) <bootloader_monotonic_counter>`, :ref:`Semantic (SW) <ug_fw_update_image_versions_mcuboot>`
      - Image swap - single primary
        Dual-slot direct-xip
 
+The table symbol can be read as follows:
+
+``*`` SMP with MCUboot is only available for Bluetooth® Low Energy and serial UART.
+(See the :ref:`ug_fw_update` page to learn about other DFU methods.)
```

Comment:
```suggestion
The ``*`` table symbol indicates that SMP with MCUboot is only available for Bluetooth® Low Energy and serial UART.
(See the :ref:`ug_fw_update` page to learn about other DFU methods.)
```

Reply:
Thanks for the suggestions, Greg, and for clarifying. I agree, and this makes sense.
But won't implement due to change in plans (so no longer applicable)